src/us/kbase/genbank/GbkQualifier.java

```diff
@@ -5,29 +5,32 @@
  * @author rsutormin
  */
 public class GbkQualifier extends GbkElement {
-	boolean is_text = false;
-	boolean is_closed = false;
-	public GbkQualifier(int line_num,String type,String value) {
-		super(line_num,type,value);
+    boolean is_text = false;
+    boolean is_closed = false;
+    public GbkQualifier(int line_num,String type,String value) {
+	super(line_num,type,value);
+    }
+    public void close() {
+	is_closed = true;
+	String val = value.toString();
+	if (val.startsWith("\"")) {
+	    if (val.endsWith("\"")) {
+		is_text = true;
+		value.deleteCharAt(0);

```

Comment:
Also, a short code comment describing why this is here should exist.  I realize you are adding code here and the original was also not commented, but it needs some sort of description in the code.